packages/app/src/app/pages/Dashboard/Content/routes/Sandboxes/EmptyState.tsx

```diff
@@ -0,0 +1,177 @@
+import { TemplateCard } from 'app/components/CreateSandbox/TemplateCard';
+import { useWorkspaceLimits } from 'app/hooks/useWorkspaceLimits';
+import { TemplateFragment } from 'app/graphql/types';
+import { useActions, useAppState } from 'app/overmind';
+
+const StyledEmptyDescription = styled(Text)`
+  margin: 0;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #999999;
+`;
+
+const TEMPLATE_IDS = [
+  'k8dsq1', // NodeJS (cloud)
+  'vanilla', // Vanilla JS (browser)
+  'uo1h0', // NextJS (cloud)
+  '9qputt', // React + Vite (browser)
+];
+
+export const EmptyState: React.FC = () => {
+  const { isPersonalSpace } = useWorkspaceAuthorization();
+  const { hasMaxPublicSandboxes } = useWorkspaceLimits();
+  const officialTemplates = useOfficialTemplates();
+  const actions = useActions();
+  const { dashboard } = useAppState();
+
+  const handleOpenTemplate = (template: TemplateFragment) => {
+    const { sandbox } = template;
+
+    const url = sandboxUrl(sandbox);
+    track('Sandboxes - open template from empty state', {
+      codesandbox: 'V1',
+      event_source: 'UI',
+    });
+
+    window.open(url, '_blank');
+  };
+
+  const handleSelectTemplate = (template: TemplateFragment) => {
+    const { sandbox } = template;
+    const collection = dashboard.allCollections?.find(c => c.path === '/');
+
+    track('Sandboxes - select template from empty state', {
+      codesandbox: 'V1',
+      event_source: 'UI',
+    });
+
+    actions.editor.forkExternalSandbox({
+      sandboxId: sandbox.id,
+      openInNewWindow: false,
+      body: {
+        collectionId: collection?.id,
+      },
+    });
+
+    actions.modals.newSandboxModal.close();
+  };
+
+  return (
+    <Stack
+      css={{
+        width: `calc(100% - ${2 * GUTTER}px)`,
+        maxWidth: GRID_MAX_WIDTH - 2 * GUTTER,
+        margin: '24px auto 0',
+      }}
+      direction="vertical"
+      gap={10}
+    >
+      {isPersonalSpace ? (
+        <StyledEmptyDescription as="p">
+          Sandboxes are a great way to prototype your ideas with zero startup
+          costs and with everything you need: a code editor, previews, dev
+          servers, unit tests, Storybook and many other devtools.
+        </StyledEmptyDescription>
+      ) : (
+        <Stack
+          css={{
+            width: '100%',
+            '@media screen and (min-width: 768px)': {
+              width: '560px',
+            },
+          }}
+          direction="vertical"
+          gap={6}
+        >
+          <StyledEmptyDescription as="p">
+            Sandboxes are a great way to prototype your ideas with zero startup
+            costs and with everything you need: a code editor, previews, dev
+            servers, unit tests, Storybook and many other devtools.
+          </StyledEmptyDescription>
+          <StyledEmptyDescription as="p">
+            Sandboxes in this section are visible to all team members.
```

Comment:
copy suggestion: use `folder` instead of `section` as you see this view in an empty folder as well.